packages/desktop-client/src/components/reports/reports/SpendingCard.tsx

```diff
@@ -34,8 +34,8 @@ export function SpendingCard() {
       : monthUtils.getDay(monthUtils.currentDay()) - 1;
   const difference =
     data &&
-    data.intervalData[todayDay].lastMonth -
-      data.intervalData[todayDay].thisMonth;
+    data.intervalData[todayDay].average - data.intervalData[todayDay].thisMonth;
```

Comment:
This should stay as lastMonth since we are using lastMonth as our conditional variable.

Reply:
This is still set to average and should be lastMonth.

Reply:
Forgot to fix this... thanks again.